app/views/document_topics/edit.html.erb

```diff
@@ -1,24 +1,39 @@
 <% content_for :back_link, document_path(@document) %>
 <% content_for :title, t("document_topics.edit.title", title: @document.title_or_fallback) %>
 
-<% def unroll(topic) %>
+<% def unroll(topic_content_id) %>
   <% capture do %>
     <li>
-      <%= @topic_index[topic][:title] %>
+      <%= tag.input type: "checkbox",
+        id: topic_content_id,
+        name: "topics[]",
+        value: topic_content_id,
+        checked: @topic_content_ids.include?(topic_content_id) %>
 
-      <% if @topic_index[topic][:children].any? %>
+      <%= @topic_index[topic_content_id][:title] %>
```

Comment:
This may be moot if this all replaced by the miller component but I'd guess we'd want this to be a label of the checkbox.

Reply:
Also done - thanks for pointing out :-). We tried to use the checkbox component, but it's not really designed for single item usage. Will leave this to @alex-ju.